<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientManagerForm</class>
 <widget class="QWidget" name="ClientManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientManagerForm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="minimumSize">
      <size>
       <width>427</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="listVerticalLayout">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Client ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Client Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Phone Number</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Email</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>178</width>
         <height>236</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="clientFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="clientIDLabel">
            <property name="text">
             <string>&amp;Client ID</string>
            </property>
            <property name="buddy">
             <cstring>clientIDLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="clientIDLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="clientNameLabel">
            <property name="text">
             <string>Client &amp;Name</string>
            </property>
            <property name="buddy">
             <cstring>clientNameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="clientNameLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="phoneNumberLabel">
            <property name="text">
             <string>&amp;Phone Number</string>
            </property>
            <property name="buddy">
             <cstring>phoneNumberLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="phoneNumberLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="addressLabel">
            <property name="text">
             <string>&amp;Address</string>
            </property>
            <property name="buddy">
             <cstring>addressLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="addressLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="emailLabel">
            <property name="text">
             <string>&amp;Email</string>
            </property>
            <property name="buddy">
             <cstring>emailLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="emailLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="addHorizontalLayout">
          <item>
           <widget class="QPushButton" name="addPushButton">
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyPushButton">
            <property name="text">
             <string>&amp;Modify</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>177</width>
         <height>246</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeWidget" name="searchTreeWidget">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Client ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Client Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Phone Number</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Address</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Email</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="searchComboBox">
            <item>
             <property name="text">
              <string>Client ID Search</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Client Name Search</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Phone Number Search</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Address Search</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Email Search</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="searchPushButton">
          <property name="text">
           <string>S&amp;earch</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>clientIDLineEdit</tabstop>
  <tabstop>clientNameLineEdit</tabstop>
  <tabstop>phoneNumberLineEdit</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>addPushButton</tabstop>
  <tabstop>modifyPushButton</tabstop>
  <tabstop>searchTreeWidget</tabstop>
  <tabstop>searchComboBox</tabstop>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>searchPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
